### DISCUSSION TITLE
Entry-Level/Junior/Beginner Frontend Developer Technical Questions

### DISCUSSION THREAD

**Discussion by YayoKB** [EddieHubCommunity/support#5897] [Tips and Tricks]
Hi all, I was lucky enough to get a sample of some technical questions for a Junior Frontend Developer interview.

### Why am I sharing this? What is it for?
As a Junior myself with no interview experience, I had nothing to go on regarding technical questions. I had no idea what would be expected of me as a Junior. So, by sharing these questions, I'm hoping you will gain an understanding of what a real company may expect a Junior Frontend Developer to know. By reading through these questions, you can figure out what you do know and what you should work on.

### HTML

- Why is HTML important?
- What is the difference between a tag and an attribute in HTML?
- What is the difference between a span and div tag? When would you use either?
- What is the role of the head tag in an HTML document?
- What is the DOM?

### CSS

- Why is CSS important?
- What is the CSS box model?
- Describe different types of selectors used in CSS
- What are different ways to hide an element with CSS?
- What is CSS grid?

### JavaScript

- What is JavaScript?
- Is JavaScript a case-sensitive language?
- What are the advantages of JavaScript?
- What are the disadvantages of JavaScript?
- What are the scopes of a variable in JavaScript?
- What is the purpose of ‘This’ operator in JavaScript?

**Comment by eddiejaoude** [comment]
Thank you for sharing Kate 👍
